package.json

```diff
@@ -173,7 +174,7 @@
 		"babel-plugin-named-asset-import": "^0.3.6",
 		"babel-preset-react-app": "^9.1.2",
 		"case-sensitive-paths-webpack-plugin": "2.3.0",
-		"cypress": "^5.0.0",
+		"cypress": "^10.6.0",
 		"cypress-cucumber-preprocessor": "^2.5.0",
```

Comment:
this should be updated to  "cypress-cucumber-preprocessor": "^4.3.1"

Also, Cypress is recommending to use a different cucumber preprocessor https://github.com/badeball/cypress-cucumber-preprocessor . I think this is the one that will be supported moving forward, please discuss it with a team if we want to make a switch now